doc/elements.md

```diff
@@ -1041,28 +1041,49 @@ The provided `script-name` corresponds to a file located within the director for
 
 -----
 
-## `pl-graphviz-render` element
+## `pl-graph` element
 
 Using the [viz.js](https://github.com/mdaines/viz.js/) library, create 
-Graphviz DOT visualizations .
+Graphviz DOT visualizations.
 
-#### Sample Element
+#### Sample Element 1
```

Comment:
Remove the 1 and 2. Switch to  Sample Elements

Use `---` to create a horizontal rule between two sections if need be.